### DISCUSSION TITLE
Revise the Getting Started docs

### DISCUSSION THREAD

**Discussion by nikkimk** [RedHat-UX/red-hat-design-system#1150] [Ideas]
@Nouveau and I were discussing the **Getting Started** docs and wondered if we need to think about the audience for the docs, as we have several audiences to consider: design system consumers versus contributors and designers versus developers.

We took a look at how other design system docs are organized:
- [Carbon](https://carbondesignsystem.com/) organizes into **Designing**, **Developing**, **Contributing**, and **Migrating**
- [Patternfly](https://www.patternfly.org/) organizes into **Contribute->Design**, **Contribute->Develop**, **Get Started->Contribute**, and **Get Started->Design and Develop** under it.
- [Patternfly elements](https://patternflyelements.org/) organizes into **Get started** and **Develop**
- [Material design]() organizes into **Get started->Design**, **Get started->Develop**, and **Develop**

We're thinking we should reorganized this content something like this:
- Get started
   - For designers
     - Overview
     - Design system kit
     - FTS
     - Icons, etc.
  - For developers
     - Overview
     - Using npm
     - Using CDN**
     - React wrappers* coming soon
     - Vue wrappers* coming soon
- Contribute***
   - Overview
   - Bugs and requests
   - Documentation
   - Components
   - Icons
   - Patterns
  
**Link to VPN for Red Hat-specific Implementation
***Contribute page* see Carbon’s contributing section

**Comment by brianferry** [comment]
I really like having the designers and the developers sections for the different info.  I'm also into the contribution section, seeing how Carbon did it makes sense just so long as we're not having to duplicate content from the repo specific readme files.  Though I can also see as we get more and more repos we'll need this general information for a lot of projects.  +1

**Comment by bennypowers** [comment]
This is great! I agree that following the upstream template is wise. The main landing page could be a good place to use a tile nav?
